frontend/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Racxob</title>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=RACXOB_ID"></script>
```

Comment:
I don't know much about this stuff, but this will result in a 404 (in the console), right? Does that hurt SEO or anything else? At the very least, it is a bit annoying when debugging.

Reply:
idk much about google analytics but fwiw i used `react-ga` for the sb.com prototype
```jsx
ReactGA.initialize('my-tracking-code')
const history = createBrowserHistory()
history.listen((location, _) => {
  ReactGA.set({ page: location.pathname })
  ReactGA.pageview(location.pathname)
})
```

Reply:
It will not. GA has written this in such a way where it's basically 100% guaranteed to never fail, even if the key is wrong

Reply:
I hesitate to add yet another library around GA, but I will add the types, I forgot about that. That way, it's still typesafe, but done by the way the GA docs recommend and outline